parser/openfoodfacts_taxonomy_parser/utils.py

```diff
@@ -29,6 +29,15 @@ def normalize_text(line: str, lang="default", char="-"):
     # Removing excess "-"
     line = re.sub(r"-+", char, line)
     line = line.strip(char)
+
+    # Remove stopwords
+    if lang in stopwords:
+        stopwords = stopwords[lang]
+        line_surrounded_by_char = char + line + char
+        for stopword in stopwords:
+            line_surrounded_by_char = line_surrounded_by_char.replace(char + stopword + char, char)
+        line = line_surrounded_by_char[1:-1]
```

Comment:
```bash
    replace_expr = "|".join(stopwords.get(lang, []))
    if replace_expr:
        # replace stopwords by "-"
        line = re.sub(r'\b(' + replace_expr + r')\b', '-', line)
```

Reply:
I've tried this and it doesn't seem to work because `-` are not considered as words